web/src/Sidebar.scss

```diff
@@ -3,21 +3,24 @@
 .Sidebar {
   width: $sidebar-width;
   max-width: $sidebar-maxWidth;
-  min-height: 100vh;
   margin-left: $spacing-unit;
   border-left: 2px solid $color-navy-light;
   box-sizing: border-box;
   position: fixed;
+  overflow-y: scroll;
+  top: 0;
+  bottom: $statusbar-height;
   right: 0;
   z-index: 900;
   background-color: $color-navy;
-  transform: translateX(120%);
+  transform: translateX(0%);
   transition: transform ease-out 200ms;
   font-size: $font-size;
 }
-.Sidebar.is-open {
-  transform: translateX(0%);
+.Sidebar.is-closed {
+  transform: translateX(100%);
   transition: transform ease-in 200ms;
+  right: $spacing-unit * 1.5;
```

Comment:
does this feel jumpy? it might be better to do something like:
transform: translateX(calc(100%  - $spacing-unit * 1.5));